docs/roman/datamodels/datamodels_asdf.rst

```diff
@@ -0,0 +1,124 @@
+.. _datamodels_asdf:
+
+
+Working with Roman datamodels and ASDF files
+============================================
+
+If you've installed the roman calibration pipeline you should also have access
+to the standalone tool asdfinfo which allows access to asdf (and roman) files
+from the terminal prompt,::
+
+    asdftool info r0000101001001001001_01101_0001_WFI16_cal.asdf
+    root (AsdfObject)
+    ├─asdf_library (Software)
+    │ ├─author (str): The ASDF Developers
+    ...
+
+also useful is::
+
+    asdftool help
+    usage: asdftool [-h] [--verbose] {help,explode,implode,extract,defragment,diff,edit,remove-hdu,info,extensions,tags,to_yaml} ...
+
+    Commandline utilities for managing ASDF files.
+
+    optional arguments:
+      -h, --help            show this help message and exit
+      --verbose, -v         Increase verbosity
+
+    subcommands:
+
+      {help,explode,implode,extract,defragment,diff,edit,remove-hdu,info,extensions,tags,to_yaml}
+        help                Display usage information
+        explode             Explode a ASDF file.
+        implode             Implode a ASDF file.
+        extract             Extract ASDF extensions in ASDF-in-FITS files into pure ASDF files
+        defragment          Defragment an ASDF file..
+        diff                Report differences between two ASDF files
+        remove-hdu          Remove ASDF extension from ASDF-in-FITS file
+        info                Print a rendering of an ASDF tree.
+        extensions          Show information about installed extensions
+        tags                List currently available tags
+        to_yaml             Convert as ASDF file to pure YAML.
+
+
+which gives a list of possible actions one of the more useful can be::
+
+    asdftool edit file.asdf
+
+Which will open the file in an editor you have set via the EDITOR environment variable.
+A more complete description of the `asdftool` can be found at _.
```

Comment:
Use double back ticks around `asdftool` to avoid attempt to link it.
The link to the page does not work for me or I don't look at it correctly.
(We need to set up the RTD build as a separate check.)
I think this should work:
```
A more complete description of the ``asdftool`` can be found at `asdftool`_.

.. _asdftool: https://asdf.readthedocs.io/en/stable/asdf/asdf_tool.html
```

Reply:
The links are fixed and are working.